docs/foundation/ui-colors.md

```diff
@@ -26,6 +26,26 @@ Pelican provides more clarity in the user interface by using a color for UI elem
       </div>
     </div>
   </div>
+  <div class="col-md-4">
+    <div class="card border-0">
+      <div class="bg-ui-light rounded-top pd-color-block"></div>
+      <div class="card-body">
+        <p class="mb-0 fw-bold">UI Light</p>
+        <p class="mb-0">Hex Code: <code>#ebeeef</code></p>
+        <p class="mb-0">SCSS Var: <code>$gray-100</code></p>
+      </div>
+    </div>
+  </div>
+  <div class="col-md-4">
+    <div class="card border-0">
+      <div class="bg-ui-priority rounded-top pd-color-block"></div>
+      <div class="card-body">
+        <p class="mb-0 fw-bold">UI Priority</p>
+        <p class="mb-0">Hex Code: <code>#004ba1</code></p>
+        <p class="mb-0">SCSS Var: <code>$info-900</code></p>
+      </div>
+    </div>
+  </div>
 </div>
 
 Like [Brand Colors](/foundation/status-colors) and [Status Colors](/foundation/status-colors), this can be changed. Change with great care as this has been chosen in accordance with usability and [Color Contrast](/accessibility/color-contrast/) guidelines.
```

Comment:
Are the other blocks of text for "Using UI Colors" missing for the new UI shades?

Reply:
it was missing before this PR